res/test1

```diff
@@ -0,0 +1,8 @@
+#Test graph 1
```

Comment:
You do not need these test files. Put them into .gitignore

Reply:
I use those files in unit tests, should I keep them?

Reply:
I think it is not necessary to include them in the github